### DISCUSSION TITLE
Please allow users to disable and hide the floating progress bar completely.

### DISCUSSION THREAD

**Discussion by zhouxinghong** [mpv-player/mpv#14122] [Q&A] [answered]
Please allow users to disable and hide the floating progress bar completely.
![image](https://github.com/mpv-player/mpv/assets/54253524/6b603a46-b6d7-4078-833e-9240409da43a)

Windows 10 | 11

**Comment by zhouxinghong** [comment]
Or how to hide the floating progress bar?

**Comment by kasper93** [comment, marked as answer]
> How to change the vertical position of the floating progress bar Or how to hide the floating progress bar?

https://mpv.io/manual/master/#options-osd-bar-align-x
```
--osd-bar-align-x=<-1-1>
    Position of the OSD bar. -1 is far left, 0 is centered, 1 is far right. Fractional values (like 0.5) are allowed.
--osd-bar-align-y=<-1-1>
    Position of the OSD bar. -1 is top, 0 is centered, 1 is bottom. Fractional values (like 0.5) are allowed.
--osd-bar-w=<1-100>
    Width of the OSD bar, in percentage of the screen width (default: 75). A value of 50 means the bar is half the screen wide.
--osd-bar-h=<0.1-50>
    Height of the OSD bar, in percentage of the screen height (default: 3.125).
--osd-bar-border-size=<size>

    Size of the border of the OSD bar in scaled pixels (see --sub-font-size for details).

    Default: 0.5.
```

> Or how to hide the floating progress bar?

https://mpv.io/manual/master/#options-osd-bar
```
--osd-bar=<yes|no>

    Enable display of the OSD bar (default: yes).

    You can configure this on a per-command basis in input.conf using osd- prefixes, see Input Command Prefixes. If you want to disable the OSD completely, use --osd-level=0.
```

**Comment by zhouxinghong** [comment]
Please allow users to disable and hide the floating progress bar completely. Since there is already a fixed progress bar at the bottom of the mpv, the floating progress bar is practically meaningless and obscures the playback screen.

For example, potplayer , vlc will not show the floating progress bar when fast forwarding and rewinding

So it is better to allow users to disable the floating progress bar completely and hide it.

![image](https://github.com/mpv-player/mpv/assets/54253524/f119e4ef-4e88-4448-8f1c-a850c9251490)

**Reply by zhouxinghong** [reply]
Since there is already a fixed progress bar at the bottom of the mpv, the floating progress bar is practically meaningless and obscures the playback screen.

**Reply by zhouxinghong** [reply]
```
Wheel left/right
Seek forward/backward 10 seconds.
```
Since there is already a fixed progress bar at the bottom of the mpv, the floating progress bar is practically meaningless and obscures the playback screen.

**Comment by zhouxinghong** [comment]
Since there is already a fixed progress bar at the bottom of the mpv, the floating progress bar is practically meaningless and obscures the playback screen.

**Comment by zhouxinghong** [comment]
![image](https://github.com/mpv-player/mpv/assets/54253524/cabb3395-d60e-4faf-acd7-2abf3c6f203a)

**Reply by zhouxinghong** [reply]
```
Wheel left/right
Seek forward/backward 10 seconds.
```
Since there is already a fixed progress bar at the bottom of the mpv, the floating progress bar is practically meaningless and obscures the playback screen.

**Comment by zhouxinghong** [comment]
![image](https://github.com/mpv-player/mpv/assets/54253524/ec77ca89-43f9-4540-a147-fa7b1f069892)

**Reply by kasper93** [reply]
Read this https://github.com/mpv-player/mpv/discussions/14122#discussioncomment-9402545 and stop asking questions that are already answered. I don't know what else do you want.
